tests/internal/src/test_cmake_format_conformant.cmake

```diff
@@ -0,0 +1,116 @@
+# Example from cmake-format documentation
```

Comment:
We should either add comments to attribute these files to their source, or develop our own test files instead.